src/event/event_queue.rs

```diff
@@ -49,19 +49,19 @@ type Callback = Box<dyn FnOnce(Env) + Send + 'static>;
 /// }
 /// ```
 
-pub struct EventQueue {
+pub struct Channel {
     tsfn: ThreadsafeFunction<Callback>,
     has_ref: bool,
 }
 
-impl std::fmt::Debug for EventQueue {
+impl std::fmt::Debug for Channel {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.write_str("EventQueue")
+        f.write_str("Channel")
     }
 }
 
-impl EventQueue {
-    /// Creates an unbounded queue for scheduling closures on the JavaScript
+impl Channel {
```

Comment:
What do you think of `Sender` to make it align even more with `std`? I think I still favor `Channel`.

Reply:
I think `Sender` only makes sense when there's also a `Receiver`. I think I prefer `Channel` as well.